### PR TITLE
[TD-11995]<feature>: support remote_read and remote_write

### PR DESCRIPTION
**Pull request by huskar-t**

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -192,13 +193,41 @@ Prometheus 使用的由*NIX内核暴露的硬件和操作系统指标的输出
 * 设置 node_exporter 的相关配置
 * 重新启动 taosAdapter
 
+### prometheus
+
+remote_read 和 remote_write 是 Prometheus 数据读写分离的集群方案。  
+只需要将 remote_read 和 remote_write url 指向 taosAdapter 对应的 url 同时设置 Basic 验证即可使用。  
+remote_read url :  http://host_to_taosAdapter:port(default 6041)/prometheus/v1/remote_read/:db  
+remote_write url :  http://host_to_taosAdapter:port(default 6041)/prometheus/v1/remote_write/:db
+
```
Comment (zitsen) [inline README-CN.md:202]: Use Markdown uordered list style here.

File: README-CN.md [hunk 2]
```diff
@@ -192,13 +193,41 @@ Prometheus 使用的由*NIX内核暴露的硬件和操作系统指标的输出
...
+Basic验证：
+username： TDengine 连接用户名  
+password： TDengine 连接密码
```
Comment (zitsen) [inline README-CN.md:205]: Use Markdown uordered list style here too.

File: README-CN.md [hunk 3]
```diff
@@ -192,13 +193,41 @@ Prometheus 使用的由*NIX内核暴露的硬件和操作系统指标的输出
...
+remote_read 和 remote_write 是 Prometheus 数据读写分离的集群方案。  
+只需要将 remote_read 和 remote_write url 指向 taosAdapter 对应的 url 同时设置 Basic 验证即可使用。  
+remote_read url :  http://host_to_taosAdapter:port(default 6041)/prometheus/v1/remote_read/:db  
+remote_write url :  http://host_to_taosAdapter:port(default 6041)/prometheus/v1/remote_write/:db
+
+Basic验证：
+username： TDengine 连接用户名  
+password： TDengine 连接密码  
+
+示例 prometheus.yml  如下：
```
Comment (zitsen) [inline README-CN.md:207]: Add line break before code .

File: README-CN.md [hunk 4]
```diff
@@ -192,13 +193,41 @@ Prometheus 使用的由*NIX内核暴露的硬件和操作系统指标的输出
...
+```
+
 ## 配置方法
 
 taosAdapter 支持通过命令行参数、环境变量和配置文件来进行配置。
 
 命令行参数优先于环境变量优先于配置文件，命令行用法是arg=val，如 taosadapter -p=30000 --debug=true，详细列表如下：
 
-```shell
+`````shell
```
Comment (zitsen) [inline README-CN.md:230]: Three ` .

File: README-CN.md [hunk 5]
```diff
@@ -267,7 +297,7 @@ Usage of taosAdapter:
       --statsd.worker int                            statsd write worker. Env "TAOS_ADAPTER_STATSD_WORKER" (default 10)
       --taosConfigDir string                         load taos client config path. Env "TAOS_ADAPTER_TAOS_CONFIG_FILE"
       --version                                      Print the version and exit
-```
+`````
```
Comment (zitsen) [inline README-CN.md:300]: Three "`" character ?

File: README.md [hunk 6]
```diff
@@ -69,7 +71,8 @@ Then you should find taosAdapter binary executable file in the working directory
 * Compatible with InfluxDB v1 write interface.  
   [https://docs.influxdata.com/influxdb/v2.0/reference/api/influxdb-1x/write/](https://docs.influxdata.com/influxdb/v2.0/reference/api/influxdb-1x/write/)
 * Compatible with opentsdb JSON and telnet format writing.  
-  [http://opentsdb.net/docs/build/html/api_http/put.html](http://opentsdb.net/docs/build/html/api_http/put.html)
+  [http://opentsdb.net/docs/build/html/api_http/put.html](http://opentsdb.net/docs/build/html/api_http/put.html)
```
Comment (zitsen) [inline README.md:74]: Recommanded bare url style in Markdown for GitHub:

```markdown
<http://opentsdb.net/docs/build/html/api_http/put.html>
```

Rule: don't repeat yourself.

File: README.md [hunk 7]
```diff
@@ -103,6 +107,7 @@ Support following InfluxDB query parameters:
 * `u` user non-essential parameters
 * `p` password Optional parameter
 
+Note: There is currently not supported token authentication in InfluxDB only supports Basic authentication and query parameter authentication.
```
Comment (zitsen) [inline README.md:110]: Add line break before headers/paragraphs/sections.

File: README.md [hunk 8]
```diff
@@ -188,6 +193,34 @@ Prometheus exporter for hardware and OS metrics exposed by *NIX kernels
 * Set the relevant configuration of node_exporter
 * Restart taosAdapter
 
+### prometheus
```
Comment (zitsen) [inline README.md:196]: Here too.

**Review by zitsen — CHANGES_REQUESTED** [review]
Please update markdown files.

File: README-CN.md (hunk 4 above)
Comment (sangshuduo) [inline README-CN.md:230]: 多了两个 ` ？

**Review by sangshuduo — COMMENTED** [review]
see my comments

File: README-CN.md (hunk 1 above)
Comment (huskar-t) [reply README-CN.md:203]: fixed

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by zitsen — APPROVED** [review]
LGTM

**Review by wu-champion — COMMENTED** [review]
LGTM

**Pull request merged**